src/components/Organisms/Todocard/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import AddTaskButton from "../../Atoms/AddTaskButton";
+import Task from "../../Molecules/Task";
+
+const TodoCard = () => {
+  const [tasklist, setTasklist] = useState([]);
+  const onAddTaskButtonClick = () => {
+    const NewTask = {
+      name: "",
+      initializing: true,
+    };
+    setTasklist(tasklist.concat(NewTask));
+  };
+  const onTaskComplete = (index) => {
+    let NewTasklist = tasklist.filter((_, i) => i !== index);
+    setTasklist(NewTasklist);
+  };
+  const onTaskNameChange = (value, index) => {
+    let NewTasklist = [...tasklist];
+    if (value === "") {
+      NewTasklist.splice(index, 1);
+    } else {
+      NewTasklist.splice(index, 1, {
+        name: value,
+        initializing: false,
+      });
+    }
+    setTasklist(NewTasklist);
+  };
+  return (
+    <StyledWrapper>
+      <AddTaskButton onClick={onAddTaskButtonClick} />
+      <StyledTaskList>
+        {tasklist.map((task, index) => (
+          <Task
+            key={index}
+            onTaskComplete={() => onTaskComplete(index)}
+            onTaskNameChange={(value) => {
+              onTaskNameChange(value, index);
+            }}
+            taskName={task.name}
+            defaultIsEditing={task.initializing}
+          ></Task>
```

Comment:
```suggestion
          <Task
            key={index}
            onTaskComplete={() => onTaskComplete(index)}
            onTaskNameChange={(value) => {
              onTaskNameChange(value, index);
            }}
            taskName={task.name}
            defaultIsEditing={task.initializing}
          />
```
子要素を持たない場合、このようにも書くことが出来ます。